yarn.lock

```diff
@@ -1902,6 +1902,16 @@ array.prototype.flat@^1.2.1, array.prototype.flat@^1.2.3:
     define-properties "^1.1.3"
     es-abstract "^1.17.0-next.1"
 
+array.prototype.flatmap@^1.2.3:
```

Comment:
huh do you know why we have so many new entries in the lock file? Not important, but I'm just curious? I would have thought this would be the same or a reduction.

Reply:
This does suggest something went wrong. I’ll look into it.